src/Spice86.Core/Emulator/CPU/InstructionsImpl/Instructions8.cs

```diff
@@ -378,7 +378,12 @@ public class Instructions8 : Instructions {
 
     public void Sahf() {
         // SAHF
-        State.Flags.FlagRegister = State.AH;
+        // EFLAGS(SF:ZF:0:AF:0:PF:1:CF) := AH;
```

Comment:
OF is not 0, it's whatever it was isnt it?
In that case, wouldnt it be EFLAGS(SF:ZF:0:AF:XX:PF:1:CF) := AH;